src/mpc-hc/AppSettings.cpp

```diff
@@ -1922,6 +1924,13 @@ void CAppSettings::UpdateRenderersData(bool fSave)
             delete [] dPtr;
         }
 
+        ars.bCacheShaders = !!pApp->GetProfileInt(IDS_R_SETTINGS, IDS_RS_CACHESHADERS, DefaultSettings.bCacheShaders);
+        if (AfxGetMyApp()->GetAppDataPath(ars.sShaderCachePath)) {

```

Comment:
Please use `GetAppSavePath()` if .ini file is used cache should be stored in the same folder as binary.